bigbluebutton-web/build.gradle

```diff
@@ -72,6 +72,10 @@ dependencies {
   implementation "org.grails.plugins:scaffolding"
   implementation "org.grails.plugins:events"
   implementation "org.grails.plugins:gsp"
+  implementation "org.apache.tomcat.embed:tomcat-embed-core:9.0.62"

```

Comment:
Could we do

`tomcat-embed-version "9.0.62"` like we do for `version`
so that next time we have to upgrade it's a 1 line change and we don't miss any dependency